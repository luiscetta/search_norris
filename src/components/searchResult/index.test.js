import React from 'react';

import { create } from 'react-test-renderer';

import SearchResults from './';

function MockedResults() {
    return <div id="results"/>;
}

describe('Search Results component', () => {
    describe('When the component is rendered', () => {
        it('Should match snapshot', () => {
            const wrapper = create(<SearchResults results={MockedResults} />);
            expect(wrapper.toJSON()).toMatchSnapshot();
        });
    });
});