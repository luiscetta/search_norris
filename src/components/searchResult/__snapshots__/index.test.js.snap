// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Results component When the component is rendered Should match snapshot 1`] = `
<div
  className="results-container"
>
  <h2>
    Search results:
  </h2>
  <div
    className="results-content"
  >
    <p>
      No results found!
    </p>
  </div>
</div>
`;
